tests/systemd_testsuite/test_01_basic.pm

```diff
@@ -18,7 +18,36 @@ use utils 'zypper_call';
 use power_action_utils 'power_action';
 
 sub run {
-    select_console 'root-console';
+    #prepare test
+    assert_script_run 'cd /var/opt/systemd-tests';
+    assert_script_run './run-tests.sh TEST-01-BASIC --setup 2>&1 | tee /tmp/testsuite.log', 600;
+    assert_script_run 'ls -l /etc/systemd/system/testsuite.service';
+    #reboot
+    power_action('reboot', textmode => 1);
+    assert_screen('linux-login', 600);
+    type_string "root\n";
+    wait_still_screen 3;
+    type_password;
+    wait_still_screen 3;
+    send_key 'ret';
+    #run test
+    assert_script_run 'cd /var/opt/systemd-tests';
+    assert_script_run './run-tests.sh TEST-01-BASIC --run 2>&1 | tee /tmp/testsuite.log', 60;
+    #cleanup
+    assert_screen("systemd-testsuite-test-01-basic");
+    assert_script_run './run-tests.sh TEST-01-BASIC --clean 2>&1 | tee /tmp/testsuite.log', 60;
```

Comment:
Since we want to use **always_rollback**, is it necessary to perform explicit cleanup? Could it be omitted?

Reply:
Indeed, the cleanup isn't necessary for openqa. I will remove it.